app/src/page/channel/view.js

```diff
@@ -103,7 +97,127 @@ class ChannelPage extends React.PureComponent {
+          </View>}
+
+        {(website_url || email || description) &&
+        <ScrollView style={channelPageStyle.aboutScroll} contentContainerStyle={channelPageStyle.aboutScrollContent}>
+          {(website_url && website_url.trim().length > 0) &&
+          <View style={channelPageStyle.aboutItem}>
+            <Text style={channelPageStyle.aboutTitle}>Website</Text>
+            <Link style={channelPageStyle.aboutText} text={website_url} href={website_url} />
+          </View>}
+
+          {(email && email.trim().length > 0) &&
+          <View style={channelPageStyle.aboutItem}>
+            <Text style={channelPageStyle.aboutTitle}>Email</Text>
+            <Link style={channelPageStyle.aboutText} text={email} href={`mailto:${email}`} />
+          </View>}
+
+          {(description && description.trim().length > 0) &&
+          <View style={channelPageStyle.aboutItem}>
+            <Text style={channelPageStyle.aboutText}>{description}</Text>
+          </View>}
+        </ScrollView>}
+      </View>
+    );
+  }
+
+  render() {
+    const {
+      fetching,
+      claimsInChannel,
+      claim,
+      navigation,
+      totalPages,
+      uri,
+      drawerStack,
+      popDrawerStack
+    } = this.props;
+    const { name, permanent_url: permanentUrl } = claim;
+
+    let thumbnailUrl, coverUrl, title;
+    if (claim && claim.value) {
+      title = claim.value.title;
+      if (claim.value.cover) {
+        coverUrl = claim.value.cover.url;
+      }
+      if (claim.value.thumbnail) {
+        thumbnailUrl = claim.value.thumbnail.url;
+      }
+    }
+
+
+    return (
+      <View style={channelPageStyle.container}>
+        <UriBar value={uri} navigation={navigation} />
+
+
+        <View style={channelPageStyle.viewContainer}>
+          <View style={channelPageStyle.cover}>
+            <Image
+              style={channelPageStyle.coverImage}
+              resizeMode={'cover'}
+              source={(coverUrl && coverUrl.trim().length > 0) ? { uri: coverUrl } : require('../../assets/default_channel_cover.png')} />
+
+            <View style={channelPageStyle.channelHeader}>
+              <Text style={channelPageStyle.channelName}>{(title && title.trim().length > 0) ? title : name}</Text>
+            </View>
+
+            <View style={channelPageStyle.avatarImageContainer}>
+              <Image
+                style={channelPageStyle.avatarImage}
+                resizeMode={'stretch'}
+                source={(thumbnailUrl && thumbnailUrl.trim().length > 0) ? { uri: thumbnailUrl } : require('../../assets/default_avatar.jpg')} />
+            </View>
+
+            <SubscribeButton style={channelPageStyle.subscribeButton} uri={uri} name={name} />
+          </View>
+
+          <View style={channelPageStyle.tabBar}>
+            <TouchableOpacity style={channelPageStyle.tab} onPress={() => this.setState({ activeTab: 'content' })}>
```

Comment:
'content' and 'about' should be constants.

Reply:
Agreed.